diff --git a/node_modules/@gouvfr/dsfr/dist/artwork/dark.svg b/node_modules/@gouvfr/dsfr/dist/artwork/dark.svg
index e4d335c..6c53dce 100644
--- a/node_modules/@gouvfr/dsfr/dist/artwork/dark.svg
+++ b/node_modules/@gouvfr/dsfr/dist/artwork/dark.svg
@@ -12,3 +12,4 @@
   <use class="fr-artwork-minor" href="#artwork-minor"/>
   <use class="fr-artwork-major" href="#artwork-major"/>
 </svg>
+<!-- Slightly increasing size so it reaches the threshold that will prevent it from being inlined by Vite. -->
diff --git a/node_modules/@gouvfr/dsfr/dist/dsfr.module.js b/node_modules/@gouvfr/dsfr/dist/dsfr.module.js
index 3566bc7..e58a87f 100644
--- a/node_modules/@gouvfr/dsfr/dist/dsfr.module.js
+++ b/node_modules/@gouvfr/dsfr/dist/dsfr.module.js
@@ -6453,9 +6453,11 @@ class HeaderLinks extends api.core.Instance {
       case api.Modes.ANGULAR:
       case api.Modes.REACT:
       case api.Modes.VUE:
+        /*
         this.warn(`header__tools-links content is different from header__menu-links content.
 As you're using a dynamic framework, you should handle duplication of this content yourself, please refer to documentation:
 ${api.header.doc}`);
+        */
         break;
 
       default:
