diff --git a/node_modules/@gouvfr/dsfr/dist/.DS_Store b/node_modules/@gouvfr/dsfr/dist/.DS_Store
new file mode 100644
index 0000000..9200916
Binary files /dev/null and b/node_modules/@gouvfr/dsfr/dist/.DS_Store differ
diff --git a/node_modules/@gouvfr/dsfr/dist/artwork/dark.svg b/node_modules/@gouvfr/dsfr/dist/artwork/dark.svg
index e4d335c..e4f4aac 100644
--- a/node_modules/@gouvfr/dsfr/dist/artwork/dark.svg
+++ b/node_modules/@gouvfr/dsfr/dist/artwork/dark.svg
@@ -12,3 +12,5 @@
   <use class="fr-artwork-minor" href="#artwork-minor"/>
   <use class="fr-artwork-major" href="#artwork-major"/>
 </svg>
+<!-- Slightly increasing size so it reaches the threshold that will prevent it from being inlined by Vite. -->
+
diff --git a/node_modules/@gouvfr/dsfr/dist/dsfr.module.js b/node_modules/@gouvfr/dsfr/dist/dsfr.module.js
index 588abee..b84d5b4 100644
--- a/node_modules/@gouvfr/dsfr/dist/dsfr.module.js
+++ b/node_modules/@gouvfr/dsfr/dist/dsfr.module.js
@@ -6226,9 +6226,11 @@ class HeaderLinks extends api.core.Instance {
       case api.Modes.ANGULAR:
       case api.Modes.REACT:
       case api.Modes.VUE:
+        /*
         this.warn(`header__tools-links content is different from header__menu-links content.
 As you're using a dynamic framework, you should handle duplication of this content yourself, please refer to documentation:
 ${api.header.doc}`);
+        */
         break;
 
       default:
